modules/programs/getmail-accounts.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, ... }:
+
+with lib;
+
+{
+  options.getmail = {
+    enable = mkEnableOption "the getmail mail retriever for this account";
+
+    destinationCommand = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "\${pkgs.maildrop}/bin/maildrop";
+      description = ''
+        Specify a command delivering the incoming mail to your maildir.
+      '';
+    };
+
+    mailboxes = mkOption {
+      type = types.listOf types.str;
+      default = [];
```

Comment:
does getmail mandate to set mailboxes ? would it make sense to set ["INBOX"] as default ?